app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerFragmentPresenter.kt

```diff
@@ -164,7 +164,13 @@ class QuestionPlayerFragmentPresenter @Inject constructor(
 
   fun onSubmitButtonClicked() {
     hideKeyboard()
-    handleSubmitAnswer(questionViewModel.getPendingAnswer(recyclerViewAssembler))
+    var answer = UserAnswer.getDefaultInstance()
+    if (recyclerViewAssembler.getPendingAnswerHandler(questionViewModel.itemList) != null) {
+      answer = questionViewModel.getPendingAnswerWithoutError(
+        recyclerViewAssembler.getPendingAnswerHandler(questionViewModel.itemList)!!
```

Comment:
Prefer not forcing non-nullability--handle it, instead. Ditto in StateFragmentPresenter.

Reply:
Got it